MapleServer2/Managers/FieldManager.cs

```diff
@@ -1170,7 +1186,12 @@ private partial class Npc
     {
     }
 
-    private abstract partial class FieldActor<T>
+    public void Dispose()
     {
+        CancellationToken.Cancel();
```

Comment:
After cancelling, you probably want to wait for the tasks to complete before disposing them.

`Tasks.WaitAll(...)`

A timeout could be used if you don't want to block indefinitely (not sure if WaitAll supports it)